crates/re_ui/src/toggle_switch.rs

```diff
@@ -18,11 +18,19 @@ fn toggle_switch_ui(ui: &mut egui::Ui, height: f32, on: &mut bool) -> egui::Resp
         let how_on = ui.ctx().animate_bool(response.id, *on);
         let visuals = ui.style().interact(&response);
         let expanded_rect = visual_rect.expand(visuals.expansion);
-        let fg_fill = visuals.bg_fill;
-        let bg_fill = visuals.text_color();
+        let fg_fill_off = visuals.bg_fill;
+        let fg_fill_on = egui::Color32::from_rgba_premultiplied(0, 128, 255, 255);
+        let fg_fill = egui::Color32::from_rgba_premultiplied(
+            lerp_u8(fg_fill_off.r(), fg_fill_on.r(), how_on),
+            lerp_u8(fg_fill_off.g(), fg_fill_on.g(), how_on),
+            lerp_u8(fg_fill_off.b(), fg_fill_on.b(), how_on),
+            lerp_u8(fg_fill_off.a(), fg_fill_on.a(), how_on),
+        );
```

Comment:
We should just add `lerp` member to `Color32`

Reply:
https://github.com/emilk/egui/pull/4627